pkg/util/testing/wrappers.go

```diff
@@ -566,3 +576,28 @@ func MakeAdmissionCheck(name string) *AdmissionCheckWrapper {
 func (ac *AdmissionCheckWrapper) Obj() *kueue.AdmissionCheck {
 	return &ac.AdmissionCheck
 }
+
+// WorkloadPriorityClassWrapper wraps a WorkloadPriorityClass.
+type WorkloadPriorityClassWrapper struct {
+	kueue.WorkloadPriorityClass
+}
+
+// MakeWorkloadPriorityClass creates a wrapper for a WorkloadPriorityClass.
+func MakeWorkloadPriorityClass(name string) *WorkloadPriorityClassWrapper {
+	return &WorkloadPriorityClassWrapper{kueue.WorkloadPriorityClass{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: name,
+		}},
+	}
+}
+
+// WorkloadPriorityValue update value of WorkloadPriorityClass.
+func (p *WorkloadPriorityClassWrapper) WorkloadPriorityValue(v int32) *WorkloadPriorityClassWrapper {
```

Comment:
```suggestion
func (p *WorkloadPriorityClassWrapper) Value(v int32) *WorkloadPriorityClassWrapper {
```

Reply:
Value couldn't be used as a function name because it is already being used in the "workloadPriorityClass" field. Instead, I renamed it "PriorityValue."